LetsDoStuff.Domain/Migrations/20201208194631_Initial.Designer.cs

```diff
@@ -0,0 +1,168 @@
+﻿// <auto-generated />
+using System;
+using LetsDoStuff.Domain;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Metadata;
+using Microsoft.EntityFrameworkCore.Migrations;
+using Microsoft.EntityFrameworkCore.Storage.ValueConversion;
+
+namespace LetsDoStuff.Domain.Migrations
```

Comment:
Since we now have migrations lets remove `context.Database.EnsureDeleted()` from `SeedTestData` so we can persist changes between application restarts. 

Actually, for the best, we should have an independent script/application for seeding DB with test data, but let's keep it simple, that'll do.